[27-Apr-2025 15:32:54 UTC] PHP Fatal error:  Could not check compatibility between App\Exceptions\Handler::render($request, App\Exceptions\Exception $e) and Illuminate\Foundation\Exceptions\Handler::render($request, Throwable $e), because class App\Exceptions\Exception is not available in /home/yaho/simowners.info/app/Exceptions/Handler.php on line 49
[27-Apr-2025 15:32:54 UTC] PHP Fatal error:  Uncaught ReflectionException: Class "App\Exceptions\Handler" does not exist in /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php:912
Stack trace:
#0 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(912): ReflectionClass->__construct('App\\Exceptions\\...')
#1 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(795): Illuminate\Container\Container->build('App\\Exceptions\\...')
#2 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(961): Illuminate\Container\Container->resolve('App\\Exceptions\\...', Array, false)
#3 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(296): Illuminate\Foundation\Application->resolve('App\\Exceptions\\...', Array, false)
#4 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(908): Illuminate\Container\Container->Illuminate\Container\{closure}(Object(Illuminate\Foundation\Application), Array)
#5 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(795): Illuminate\Container\Container->build(Object(Closure))
#6 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(961): Illuminate\Container\Container->resolve('Illuminate\\Cont...', Array, true)
#7 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(731): Illuminate\Foundation\Application->resolve('Illuminate\\Cont...', Array)
#8 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(946): Illuminate\Container\Container->make('Illuminate\\Cont...', Array)
#9 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(288): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#10 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(207): Illuminate\Foundation\Bootstrap\HandleExceptions->getExceptionHandler()
#11 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(189): Illuminate\Foundation\Bootstrap\HandleExceptions->renderForConsole(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#12 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(231): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#13 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(255): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#14 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}()
#15 {main}

Next Illuminate\Contracts\Container\BindingResolutionException: Target class [App\Exceptions\Handler] does not exist. in /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php:914
Stack trace:
#0 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(795): Illuminate\Container\Container->build('App\\Exceptions\\...')
#1 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(961): Illuminate\Container\Container->resolve('App\\Exceptions\\...', Array, false)
#2 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(296): Illuminate\Foundation\Application->resolve('App\\Exceptions\\...', Array, false)
#3 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(908): Illuminate\Container\Container->Illuminate\Container\{closure}(Object(Illuminate\Foundation\Application), Array)
#4 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(795): Illuminate\Container\Container->build(Object(Closure))
#5 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(961): Illuminate\Container\Container->resolve('Illuminate\\Cont...', Array, true)
#6 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(731): Illuminate\Foundation\Application->resolve('Illuminate\\Cont...', Array)
#7 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(946): Illuminate\Container\Container->make('Illuminate\\Cont...', Array)
#8 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(288): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#9 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(207): Illuminate\Foundation\Bootstrap\HandleExceptions->getExceptionHandler()
#10 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(189): Illuminate\Foundation\Bootstrap\HandleExceptions->renderForConsole(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#11 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(231): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#12 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(255): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#13 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}()
#14 {main}
  thrown in /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php on line 914
[27-Apr-2025 15:33:01 UTC] PHP Fatal error:  Could not check compatibility between App\Exceptions\Handler::render($request, App\Exceptions\Exception $e) and Illuminate\Foundation\Exceptions\Handler::render($request, Throwable $e), because class App\Exceptions\Exception is not available in /home/yaho/simowners.info/app/Exceptions/Handler.php on line 49
[27-Apr-2025 15:33:01 UTC] PHP Fatal error:  Uncaught ReflectionException: Class "App\Exceptions\Handler" does not exist in /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php:912
Stack trace:
#0 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(912): ReflectionClass->__construct('App\\Exceptions\\...')
#1 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(795): Illuminate\Container\Container->build('App\\Exceptions\\...')
#2 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(961): Illuminate\Container\Container->resolve('App\\Exceptions\\...', Array, false)
#3 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(296): Illuminate\Foundation\Application->resolve('App\\Exceptions\\...', Array, false)
#4 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(908): Illuminate\Container\Container->Illuminate\Container\{closure}(Object(Illuminate\Foundation\Application), Array)
#5 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(795): Illuminate\Container\Container->build(Object(Closure))
#6 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(961): Illuminate\Container\Container->resolve('Illuminate\\Cont...', Array, true)
#7 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(731): Illuminate\Foundation\Application->resolve('Illuminate\\Cont...', Array)
#8 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(946): Illuminate\Container\Container->make('Illuminate\\Cont...', Array)
#9 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(288): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#10 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(207): Illuminate\Foundation\Bootstrap\HandleExceptions->getExceptionHandler()
#11 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(189): Illuminate\Foundation\Bootstrap\HandleExceptions->renderForConsole(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#12 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(231): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#13 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(255): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#14 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}()
#15 {main}

Next Illuminate\Contracts\Container\BindingResolutionException: Target class [App\Exceptions\Handler] does not exist. in /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php:914
Stack trace:
#0 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(795): Illuminate\Container\Container->build('App\\Exceptions\\...')
#1 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(961): Illuminate\Container\Container->resolve('App\\Exceptions\\...', Array, false)
#2 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(296): Illuminate\Foundation\Application->resolve('App\\Exceptions\\...', Array, false)
#3 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(908): Illuminate\Container\Container->Illuminate\Container\{closure}(Object(Illuminate\Foundation\Application), Array)
#4 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(795): Illuminate\Container\Container->build(Object(Closure))
#5 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(961): Illuminate\Container\Container->resolve('Illuminate\\Cont...', Array, true)
#6 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(731): Illuminate\Foundation\Application->resolve('Illuminate\\Cont...', Array)
#7 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(946): Illuminate\Container\Container->make('Illuminate\\Cont...', Array)
#8 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(288): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#9 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(207): Illuminate\Foundation\Bootstrap\HandleExceptions->getExceptionHandler()
#10 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(189): Illuminate\Foundation\Bootstrap\HandleExceptions->renderForConsole(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#11 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(231): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#12 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(255): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#13 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}()
#14 {main}
  thrown in /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php on line 914
[27-Apr-2025 15:33:04 UTC] PHP Fatal error:  Could not check compatibility between App\Exceptions\Handler::render($request, App\Exceptions\Exception $e) and Illuminate\Foundation\Exceptions\Handler::render($request, Throwable $e), because class App\Exceptions\Exception is not available in /home/yaho/simowners.info/app/Exceptions/Handler.php on line 49
[27-Apr-2025 15:33:04 UTC] PHP Fatal error:  Uncaught ReflectionException: Class "App\Exceptions\Handler" does not exist in /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php:912
Stack trace:
#0 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(912): ReflectionClass->__construct('App\\Exceptions\\...')
#1 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(795): Illuminate\Container\Container->build('App\\Exceptions\\...')
#2 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(961): Illuminate\Container\Container->resolve('App\\Exceptions\\...', Array, false)
#3 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(296): Illuminate\Foundation\Application->resolve('App\\Exceptions\\...', Array, false)
#4 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(908): Illuminate\Container\Container->Illuminate\Container\{closure}(Object(Illuminate\Foundation\Application), Array)
#5 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(795): Illuminate\Container\Container->build(Object(Closure))
#6 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(961): Illuminate\Container\Container->resolve('Illuminate\\Cont...', Array, true)
#7 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(731): Illuminate\Foundation\Application->resolve('Illuminate\\Cont...', Array)
#8 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(946): Illuminate\Container\Container->make('Illuminate\\Cont...', Array)
#9 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(288): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#10 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(207): Illuminate\Foundation\Bootstrap\HandleExceptions->getExceptionHandler()
#11 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(189): Illuminate\Foundation\Bootstrap\HandleExceptions->renderForConsole(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#12 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(231): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#13 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(255): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#14 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}()
#15 {main}

Next Illuminate\Contracts\Container\BindingResolutionException: Target class [App\Exceptions\Handler] does not exist. in /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php:914
Stack trace:
#0 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(795): Illuminate\Container\Container->build('App\\Exceptions\\...')
#1 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(961): Illuminate\Container\Container->resolve('App\\Exceptions\\...', Array, false)
#2 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(296): Illuminate\Foundation\Application->resolve('App\\Exceptions\\...', Array, false)
#3 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(908): Illuminate\Container\Container->Illuminate\Container\{closure}(Object(Illuminate\Foundation\Application), Array)
#4 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(795): Illuminate\Container\Container->build(Object(Closure))
#5 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(961): Illuminate\Container\Container->resolve('Illuminate\\Cont...', Array, true)
#6 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php(731): Illuminate\Foundation\Application->resolve('Illuminate\\Cont...', Array)
#7 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(946): Illuminate\Container\Container->make('Illuminate\\Cont...', Array)
#8 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(288): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#9 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(207): Illuminate\Foundation\Bootstrap\HandleExceptions->getExceptionHandler()
#10 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(189): Illuminate\Foundation\Bootstrap\HandleExceptions->renderForConsole(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#11 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(231): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#12 /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(255): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#13 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}()
#14 {main}
  thrown in /home/yaho/simowners.info/vendor/laravel/framework/src/Illuminate/Container/Container.php on line 914
[01-Aug-2025 10:42:12 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[01-Aug-2025 10:49:14 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[01-Aug-2025 10:49:18 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[01-Aug-2025 10:51:23 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[01-Aug-2025 10:51:26 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[01-Aug-2025 11:37:04 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[02-Aug-2025 13:40:56 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[02-Aug-2025 13:54:44 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[02-Aug-2025 14:07:44 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[02-Aug-2025 14:10:30 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[02-Aug-2025 14:21:53 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[02-Aug-2025 14:30:58 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[02-Aug-2025 14:34:22 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[02-Aug-2025 14:39:12 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[02-Aug-2025 14:40:06 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[02-Aug-2025 14:50:08 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[02-Aug-2025 14:50:46 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
[02-Aug-2025 18:06:29 UTC] PHP Warning:  Module "fileinfo" is already loaded in Unknown on line 0
